crates/polars-plan/src/dsl/mod.rs

```diff
@@ -1045,16 +1045,7 @@ impl Expr {
     #[cfg(feature = "is_in")]
     pub fn is_in<E: Into<Expr>>(self, other: E) -> Self {
         let other = other.into();
-        let has_literal = has_root_literal_expr(&other);
```

Comment:
This produced the wrong length and didn't keep the left sides name. We must go through the engine.